aruodas/spiders/aruodascrawl_rent.py

```diff
@@ -0,0 +1,34 @@
+import scrapy
+
+class aruodascrawl(scrapy.Spider):
+    name='aruodascrawl_nuoma'
+    start_urls=['https://m.en.aruodas.lt/butu-nuoma/kaune/']
+    handle_httpstatus_list = [403]
+    user_agent = "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.1 (KHTML, like Gecko) Chrome/22.0.1207.1 Safari/537.1"
```

Comment:
the user agent is used at various places. Write it in a `config.py` file that you'll import wherever needed 
```from config import USER_AGENT```